www/docs/style/sass/pages/_legacy.scss

```diff
@@ -32,33 +38,56 @@
 
     .row {
         padding-top: 0.5em;
+        margin-bottom: 1.1em;
         span.label {
             padding-right: 1em;
-        }
-        span.formw {
-            small {
-                clear: both;
-                float: left;
+            label {
+                font-size: 1em;
             }
         }
+
         @media (min-width: $medium-screen) {
             span.label {
                 display: block;
-                float: left;
                 width: 10.5em;
+                margin-bottom: 0.16667em;
             }
             span.formw {
                 display: block;
-                float: left;
                 width: 15em;
                 text-align: left;
-                small {
-                    float: none;
-                    clear: none;
+            }
+        }
+    }
+    
+    .join-form {
+        .row {
+            margin-bottom: 1.1em;
+
+            label {
+                margin-bottom: -.8em;
+            }
+
+            label[for="optintrue"], label[for="optinfalse"], label[for="mp_alerttrue"], label[for="mp_alertfalse"] {
+                display: inline;
+                font-size: inherit;
+                margin-left: .5em;
+            }
+
+            @media (min-width: $medium-screen) {
+                label {
+                    margin-bottom: .5em;
                 }
             }
         }
     }
+
+    .password-form {
+        label[for="em"] {
+            font-size: inherit;
+            margin-bottom: .5em;
```

Comment:
Feeling like we should keep the `0.` part of these just to keep to the house style